db/migrate/20181121134419_add_slug_to_articles.rb

```diff
@@ -0,0 +1,6 @@
+class AddSlugToArticles < ActiveRecord::Migration[5.2]
+  def change
+    add_column :articles, :slug, :string
```

Comment:
After you have merged this in and updated all the article slugs in production, you will want to create another release with a migration to make the slug column not null